client/stylesheets/components/_message.scss

```diff
@@ -1,7 +1,7 @@
 .info {
-  background-color: $green;
+  background-color: $message-info;
 }
 
 .error {
-  background-color: $red;
+  background-color: $message-error;
```

Comment:
I find this name ambigous, I would use it for a font color.

Reply:
Didn't understand.

Reply:
Done.